iot_api/user_api/resources/NotificationPreferences.py

```diff
@@ -185,6 +193,13 @@ def put(self):
                     raise Error.BadRequest('Asset importance name must be one these: high, medium, low. But it\'s: {0}'.format(attr))
                 setattr(nai, attr, importance.get('enabled'))
 
+            # Update asset tags
+            asset_tags = parsed_result.get('asset_tags')
+            tag_id_list = []
+            for tag in asset_tags:
+                tag_id_list.append(tag.get('id'))
```

Comment:
Here list comprehension could be used also.

Reply:
Done